docs/reference/postgresql12_migration.md

```diff
@@ -0,0 +1,133 @@
+
+Review PostgreSQL documentation to select the best option for your environment.
+
+💡 Need assistance? Connect with the Percona Everest community or consult a Percona Expert. connect with [Percona Everest community](https://forums.percona.com/t/welcome-to-perconas-community-forum/7){:target="_blank"}! Alternatively, you can always [Talk to a Percona Expert](../get-help.md#percona-experts).
+
+### Logical Dump and Restore vs. Logical Replication
+
+Two commonly used approaches to migrating a PostgreSQL database are Logical Dump and Restore and Logical Replication. Both methods help move data from one instance to another, but they have different purposes and distinct advantages.
+
+| **Feature**              | **Logical Dump and Restore**               | **Logical Replication**               |
+|--------------------------|-------------------------------------------|--------------------------------------|
+| **Setup complexity**      | Simple and portable                      | More complex, involves replication slots |
+| **Best suited for**           | One-time migrations, backups, and moving datasets across versions | Continuous data synchronization |
+| **Downtime Impact**      | Requires application downtime during migration | Minimal downtime |
+| **Primary Keys requirement** | No primary keys required; exports full table data regardless of constraints | Requires primary keys (or unique indexes) on tables|
+| **Schema compatibility**  | Allows schema modifications before restoration; supports cross-version migration | Requires schema compatibility between source and target databases |
+| **Performance**          | Can be slow for large datasets due to full export/import | More efficient for continuous updates, but may add replication overhead |
+
+### Logical Dump and Restore
+
+This option is recommended for **smaller databases** and **one-time migrations**.
+{.power-number}
+
+1. Provision a new PostgreSQl database in Percona Everest.
+
+
+
+2. Perform a logical dump of the old database:
+
+    - Use [pg_dump or pg_dumpall](https://www.postgresql.org/docs/current/app-pgdump.html) to **export** your data from PostgreSQL 12.
+
+        ??? example "Example: Source database (PostgreSQL 12)"
+            Run the following command on the source database (PostgreSQL 12):
+
+            ```sh
+            pg_dump -Fc -h <old-db-host> -U <user> <db_name> -f dump_file.dump
+            ```
+
+3. Restore data to the new Percona Everest database:
+
+    - Use [pg_restore](https://www.postgresql.org/docs/current/app-pgrestore.html) or [psql](https://www.postgresql.org/docs/current/app-psql.html) to import data into the newly created Percona Everest cluster.
+
+        ??? example "Example: Target database"
+            Run the following command on the target PostgreSQL database:
+        
+            ```sh
+            pg_restore -U your_user -h new_db_host -d new_db_name -F c your_db_backup.dump
```

Comment:
Since in the previous command you named it `dump_file.dump`
```suggestion
            pg_restore -U your_user -h new_db_host -d new_db_name -F c dump_file.dump
```